tests/e2e/subnet/suite.go

```diff
@@ -65,10 +65,10 @@ var _ = ginkgo.Describe("[Subnet]", func() {
 
 		deployOutput = commands.DeploySubnetLocally(subnetName)
 		rpcs, err = utils.ParseRPCsFromOutput(deployOutput)
-		if err != nil {
+		if err == nil {
 			fmt.Println(deployOutput)
 		}
-		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(err).Should(gomega.HaveOccurred())
```

Comment:
I don't understand why the pipeline passed before and passed now. Does the error occur or does it not?

Reply:
I changed ParseRPCsFromOutput following a previous PR comment, to return error if subnet was already deployed